lib/ui/localization/export.dart

```diff
@@ -106,11 +106,11 @@ see your mnemonic will most surely end in them stealing your account and all
 your funds so proceed with caution.
 """;
   String get securityLoginBiometrics => """
-If you understand the risks of viewing your mnemonic, please check the below
+If you understand the risks of viewing your mnemonic, please select the below
 checkbox and tap below to start the authentication process.
 """;
   String get securityLoginPassword => """
-If you understand the risks of viewing your mnemonic, please check the below
+If you understand the risks of viewing your mnemonic, please select the below
```

Comment:
please select the **checkbox below** *is more natural* then please select the **below checkbox**